testsuite/performance/tests/src/test/scala/keycloak/DefaultSimulation.scala

```diff
@@ -1,30 +1,18 @@
 package keycloak
 
-import java.util.concurrent.atomic.AtomicInteger
-
 import io.gatling.core.Predef._
-import io.gatling.core.pause.Normal
-import io.gatling.core.session._
-import io.gatling.core.validation.Validation
 import io.gatling.http.Predef._
-import org.jboss.perf.util.Util
-import org.keycloak.adapters.spi.HttpFacade.Cookie
-import org.keycloak.gatling.AuthorizeAction
-import org.keycloak.gatling.Predef._
+import keycloak.DefaultScenarioBuilder._
+
 import org.keycloak.performance.TestConfig
 
+
 /**
   * @author Radim Vansa &lt;rvansa@redhat.com&gt;
   * @author Marko Strukelj &lt;mstrukel@redhat.com&gt;
   */
 class DefaultSimulation extends Simulation {
```

Comment:
Should we also rename the the `DefaultSimulation` class (and `DefaultScenarioBuilder`) to something more descriptive, like: `OIDCLoginLogoutSimulation`, `BasicOIDCSimulation`, or maybe just `OIDCSimulation`?

Reply:
BasicOIDCSimulation sounds good to me. I'll change it.

Reply:
Can you please also update class names in the READMEs?